src/components/Header/AppSwitch/AppSwitch.module.scss

```diff
@@ -0,0 +1,23 @@
+@use '@stihl-design-system/components/styles/scss/ds';
+
+.app-switch-list {
```

Comment:
could add the comments from our header templates to the classes in this file (if there are any)